gazebo_plugins/src/gazebo_ros_camera_utils.cpp

```diff
@@ -277,16 +281,8 @@ void GazeboRosCameraUtils::LoadThread()
   this->camera_info_manager_.reset(new camera_info_manager::CameraInfoManager(
           *this->rosnode_, this->camera_name_));
 
-  // resolve tf prefix
```

Comment:
We did not. https://github.com/ros-simulation/gazebo_ros_pkgs/blame/indigo-devel/gazebo_plugins/src/gazebo_ros_camera_utils.cpp#L286